ci/set_numpy_pandas_integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+echo "Upgrading to  Nightly build of numpy pandas"
+
+PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
+
+pip install --pre --no-deps --upgrade --timeout=180 --no-cache-dir -f $PRE_WHEELS numpy pandas
```

Comment:
So this is one way we could do this, but this, but I think this is very unstable for mulitple reasons:

- we don't install dependencies of numpy and pandas (but instead the packages from the pinning)
- we might break version constraints of other packages (like dask) that sometimes constraint the upper bounds for numpy/pandas due to known problems

I think we should rather incooperate the pre-releases into the pinning process (pip-tools can pin pre-releases, but I'm not sure if it can select multiple package sources). Can you please try and figure out if that would work?

Reply:
ya, will check on that @crepererum